### DISCUSSION TITLE
Using walletConnect to connect another blockchain(rainbowKit) while enabling walletConnect in use-Wallet cause error of `Failed to execute 'define' on 'CustomElementRegistry': the name "wcm-button" has already been used with this registry`

### DISCUSSION THREAD

**Discussion by jingxuan98** [WalletConnect/walletconnect-monorepo#3383] [Developer Support]
**Describe the bug**
I am using walletConnect to connect another blockchain via (rainbowKit with wallet-connect option) while enabling walletConnect in use-Wallet for algorand (https://github.com/TxnLab/use-wallet) as well

**SDK Version (if relevant)**
    "@walletconnect/modal-sign-html": "2.5.9", // tried 2.6.0 but still same using 2.5.9 because wagmi using 2.5.9 for modal
    "wagmi": "^1.3.9",
    "@rainbow-me/rainbowkit": "^1.0.7",

**To Reproduce**
1. Setup rainbowkit
2. Setup another wallet library using wallet-connect, eg. use-Wallet for algorand
3. start the dev web

**Expected behavior**
Should be able to wallet-connect on both chains successfully

**Screenshots**
![Screenshot 2023-07-24 at 6 00 17 PM](https://github.com/WalletConnect/walletconnect-monorepo/assets/43902263/4a94afb6-9e4c-49b3-bacd-766ef9132385)

**Additional context**

**Comment by glitch-txs** [comment]
Hi, make sure you set up the SDK configuration outside the react comoponents. I think they have already an example on their repo :)

**Reply by Charl00** [reply]
Hi @glitch-txs 

I think it might be helpful to add the "their" and "the SDK" context please. 🙏 

I have the same issue and moved my configuration of walletConnect outside the component into a context and the issue persists. I looked at the **1** other link referencing the same issue and I don't think I have duplicate components. I noticed you closed that one too.

I have separate components but one, being rainbowkit wallet modal, and the other being another blockchain's wallet modal. 

In both components I need to use WalletConnect. This same error we see above appears right after clicking the WC option, configured for the second selector component only.
